src/System.IO.Ports/src/System/IO/Ports/SerialPort.cs

```diff
@@ -643,7 +648,8 @@ public int Read(byte[] buffer, int offset, int count)
                 _readPos += bytesReadToBuffer;
                 if (bytesReadToBuffer == count)
                 {
-                    if (_readPos == _readLen) _readPos = _readLen = 0;  // just a check to see if we can reset buffer
+                    if (_readPos == _readLen)
```

Comment:
braces around if body (lack of braces are only allowed for argument validation or if all ifs don't use braces)